### ISSUE TITLE
Don't use the shortcode

### ISSUE THREAD

**Issue by JoeyStk** [digitalfabrik/wp-integreat-plugin#42]
The primary issue with PHP's short tags is that PHP managed to choose a tag (<?) that was used by another syntax: XML. This is a big issue when you consider how common XML parsing and management is.

We know as of PHP 5.4, <?= ... ?> tags are supported everywhere, regardless of short tags settings. This should mean they're safe to use in portable code but in reality that has proven not to be the case. Add on to that the fact that many codesniffers won't detect lack of escaping in code when you use short-tags, and it becomes not worth the headache for anyone.

Basically the risk here is way higher than the benefits, which is why we don't permit their use.

Example(s) from your plugin:

wp-integreat-plugin/functions/integreat_shortcode.php:66: <label><b><?= $options['notification'] </b></label>
wp-integreat-plugin/functions/integreat_shortcode.php:83: <label><b><?= $options['notification'] </b></label>
wp-integreat-plugin/functions/integreat_setting.php:135: <?= $options['headline']
wp-integreat-plugin/functions/integreat_setting.php:146: <?= $options['paragraph']
